Header header
int32 board_id # board_id is begin from 1
float32[] data # xyz data of 16 tacxels [x1,y1,z1,x2,y2,z2,...,x16,y16,z16]
float32 f_n # Normal Force of all sensors
float32[] f_t # Tangential Force vector 
float32[] center_of_pressure # Center of Pressure vector
